datasets/bgt/custom_points_2386_9702.csv

```diff
@@ -0,0 +1,22 @@
+Type,X,Y
```

Comment:
Is er ergens uitgelegd hoe we deze creeren?

Reply:
Niet echt, alleen "This contains <x,y> coordinates of pole-like objects and trees. These were collected manually from different sources, but can also be scraped from PDOK." in de datasets/README

Reply:
Is daar meer info nodig denk je? Of laten we dat even tot ik de PDOK scraper af heb?

Reply:
Ja niet voor nu idd. Voor als we aan de PDOK scraper gaan werken